physical/postgresql/postgresql.go

```diff
@@ -301,6 +317,38 @@ func connectionURL(conf map[string]string) string {
 	return connURL
 }
 
+func doRetryConnect(logger log.Logger, connURL string, retries uint64) (*sql.DB, error) {
+	db, err := sql.Open("pgx", connURL)
+	if err != nil {
+		return nil, fmt.Errorf("failed to connect to postgres: %w", err)
```

Comment:
```suggestion
		return nil, err
```

else we get `failed to connect to postgres: failed to connect to postgres: ...`

Reply:
Ahh yes, thank you!